src/assets/news/2024-06-07 - Lemmy Release v0.19.4 - Image Proxying and Federation improvements.md

```diff
@@ -0,0 +1,119 @@
+
+### Media
+
+There is a new functionality for users to [list all images they have previously uploaded](https://github.com/LemmyNet/lemmy/pull/4509), and delete them if desired. It also allows admins to view and delete images hosted on the local instance.
+
+When uploading a new avatar or banner, the old one is [automatically deleted](https://github.com/LemmyNet/lemmy/pull/4573).
+
+Instance admins should also checkout [lemmy-thumbnail-cleaner](https://github.com/wereii/lemmy-thumbnail-cleaner) which can delete thumbnails for old posts, and free significant amounts of storage.
+
+### Federation
+
+Lemmy can now federate with [Wordpress](https://github.com/Automattic/wordpress-activitypub), [Discourse](https://github.com/discourse/discourse-activity-pub) and [NodeBB](https://github.com/NodeBB/NodeBB). So far there was only minor testing and these projects are still under heavy development. If you encounter any issues federating with these platforms, open an issue either in the Lemmy repo or in the respective project's issue tracker. You can test it by fetching the following posts:
+
+- [Wordpress](https://pfefferle.org/hello-lemmy-part2/)
+- [Discourse](https://socialhub.activitypub.rocks/ap/object/1899f65c062200daec50a4c89ed76dc9)
+- [NodeBB](https://community.nodebb.org/post/98325)
+
+In order to improve interoperability with Mastodon and other microblogging platforms, Lemmy now [automatically includes a hashtag with new posts](https://github.com/LemmyNet/lemmy/pull/4398). The hashtag is based on the community name, so posts to `/c/lemmy` will automatically have the hashtag `#lemmy`. This makes Lemmy posts much easier to discover.
+
+Reliability and security of federation have been improved, and numerous bugs squashed. Signed fetch was broken and is fixed now.
+
+### Vote display user setting
+
+There is now a [user setting to change the way vote counts are displayed](https://github.com/LemmyNet/lemmy/pull/4450), called [vote display mode](https://join-lemmy.org/api/interfaces/LocalUserVoteDisplayMode.html).
+
+You can specify which of the following vote data you'd like to see (or hide): Upvotes, Downvotes, Score, Upvote Percentage, or none of the above. The default (based on user feedback) is showing the upvotes + downvotes.
+
+App developers will need to update their apps to support this setting.
+
+### RSS Feeds
+
+RSS feeds now include [post thumbnail](https://github.com/LemmyNet/lemmy/pull/4413) and [embedded images](https://github.com/LemmyNet/lemmy/pull/4442).
+
+### Security Audit
+
+A security audit was recently performed on Lemmy. Big thanks to [Radically Open Security](https://www.radicallyopensecurity.com/) for the generous funding, and to Sabrina Deibe and Joe Neeman for carrying out the audit. The focus was on federation logic, and discovered various problems in this area. Most of the problems are being mitigated as part of this release. Fortunately no critical security vulnerabilities were discovered.
```

Comment:
Are there any fedi profiles for Sabrina Deibe and Joe Neeman we could link to?

Reply:
Not that I know of, but we're linking the ROS site.